qa/ccs-common-rest/build.gradle

```diff
@@ -41,6 +41,7 @@ tasks.named("yamlRestTest") {
       'search.aggregation/50_filter/Standard queries get cached',
       'search.aggregation/50_filter/Terms lookup gets cached', // terms lookup by "index" doesn't seem to work correctly
       'search.aggregation/70_adjacency_matrix/Terms lookup', // terms lookup by "index" doesn't seem to work correctly
+      'search/350_point_in_time/point-in-time with index filter'
```

Comment:
yeah, looking at the tests, its always CCS that is failing this way. Does this also bypass the `RcsCcs*` runner?

Reply:
yes it does, I think. It is not the best way to mute but it is quick and simple. I am going to look at what is up and fix it anyways.